src/Changelog/ReleaseChangelogEvent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Changelog;
+
+use Laminas\AutomaticReleases\Git\Value\BranchName;
+use Laminas\AutomaticReleases\Git\Value\SemVerVersion;
+use Laminas\AutomaticReleases\Github\Api\GraphQL\Query\GetMilestoneChangelog\Response\Milestone;
+use Laminas\AutomaticReleases\Github\Value\RepositoryName;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ReleaseChangelogEvent
```

Comment:
This should be marked with `/** @psalm-immutable */`

In addition to that, `InputInterface` and `OutputInterface` should not be part of it, as they are part of the application layer of this system, and mutable objects too.

Overall, this object seems to contain too much state that obfuscates the real dependency of consumers of it: splitting it into parameters will probably highlight some areas with too much complexity in them.